src/Chainweb/Pact/PactService.hs

```diff
@@ -627,10 +627,12 @@ execLocal
     => ChainwebTransaction
     -> Bool
       -- ^ preflight flag
+    -> Bool
```

Comment:
nit: A bool arg with a negative meaning might get confusing. Worth adding an enum `data SigVerification = On | Off`?

Reply:
Yeah... i've been thinking about whether a negative param makes sense here and I tend to agree. It makes the API a little awkward. I'll do a pass to refactor today.

Reply:
Addressed in later commits

Reply:
@imalsogreg We're on the same page! I suggested this to Emily yesterday:
```
data SigVerification = NoVerifySigs | VerifySigs
```
to put some glasses on all of this boolean blindness floating around.